app/dashboards/category_dashboard.rb

```diff
@@ -0,0 +1,58 @@
+ require "administrate/base_dashboard"
+
+class CategoryDashboard < Administrate::BaseDashboard
+  # ATTRIBUTE_TYPES
+  # a hash that describes the type of each of the model's fields.
+  #
+  # Each different type represents an Administrate::Field object,
+  # which determines how the attribute is displayed
+  # on pages throughout the dashboard.
+  ATTRIBUTE_TYPES = {
+      id: Field::Number,
+      name: Field::String,
+      short_summary: Field::String,
+      summary: Field::String,
+      parent: Field::BelongsTo.with_options(class_name: "Category"),
+      children: Field::HasMany.with_options(class_name: "Category"),
```

Comment:
Can we also show the sections?
e.g. add `sections: Field::HasMany`, and add `:sections` to `SHOW_PAGE_ATTRIBUTES` ?